PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp

```diff
@@ -75,9 +75,9 @@ bool findStrNoCase(const generic_string & strHaystack, const generic_string & st
 	return (it != strHaystack.end());
 }
 
-bool PluginsAdminDlg::isFoundInAvailableListFromIndex(int index, const generic_string& str2search, bool inWhichPart) const
+bool PluginsAdminDlg::isFoundInAvailableListFromIndex(const PluginViewList& inWhichList, int index, const generic_string& str2search, bool inWhichPart) const
```

Comment:
Nice PR.
So it won't be `isFoundInAvailableListFromIndex` but `isFoundInListFromIndex`.